source/Ship.cpp

```diff
@@ -2299,10 +2304,25 @@ bool Ship::IsDestroyed() const
 
 
 
-// Recharge and repair this ship (e.g. because it has landed).
+void Ship::Land()
+{
+	hasLanded = true;
```

Comment:
This doesnt make mission escorts that landed to refuel disapear, right?

Also since you seem to check for haslanded like you check for destroyed we should prob change it to a Removed() function?